radix-engine/src/system/system_callback.rs

```diff
@@ -82,6 +86,65 @@ pub struct SystemConfig<C: SystemCallbackObject> {
     pub modules: SystemModuleMixer,
 }
 
+pub fn load_blueprint_definition<Y, C>(
```

Comment:
Feels to me that this should still be in `system.rs` as it's part of the "service" the system provides. Is there a reason to move it here?

Reply:
It was to make it usable by both `system.rs` and `system_call.rs`, as it's now purely a utility function.

Happy to move to `SystemService` (seems like another centralised place for such functionalities).

Reply:
Moved to `SystemService`.

All I need to do is to construct a `SystemService::new(api)`. No idea why I didn't do that initially.